snapshot.py

```diff
@@ -0,0 +1,44 @@
+import json
+import subprocess
+
+def establish_ssh_connectivity():
+    node_ip_address_list = []
+    command = ["oc", "get", "nodes", "-o", "json"]
+    node_output=subprocess.check_output(command)
+
+    node_map=json.loads(node_output)
+    for item in node_map["items"]:
+        if "node-role.kubernetes.io/control-plane" in item["metadata"]["labels"]:
+            node_ip_address_list.append(item["status"]["addresses"][0]["address"])
+
+    for node_ip_address in node_ip_address_list:
+        command = ["ssh", "-o", "StrictHostKeyChecking=no", f"core@{node_ip_address}", "true"]
+        print(subprocess.run(command))
+
+
+def get_cp_names():
+    node_name_list = []
+    command = ["oc", "get", "nodes", "-o", "json"]
+    node_output=subprocess.check_output(command)
```

Comment:
We have host.py which provides functions to wrap running a subprocess that we could use here.

Reply:
i.e. host.ssh_connect()